sox.features.js

```diff
@@ -181,19 +181,16 @@
       // Description: For highlighting the names of answerers on comments
 
       function color() {
-        const answerCells = [...document.getElementsByClassName('answercell')];
-        for (let i = 0; i < answerCells.length; i++) {
-          const cell = answerCells[i];
-          const answererID = cell.querySelector('.post-signature:nth-last-of-type(1) a').href.match(/\d+/)[0];
-          const comments = document.getElementsByClassName('comments')[i + 1];
-          const commentUsers = $(comments).find('.comment-user');
-
-          for (let c = 0; c < commentUsers.length; c++) {
-            if (commentUsers[c].href.contains(`users/${answererID}`)) {
-              commentUsers[c].classList.add('sox-answerer');
-            }
-          }
-        }
+        $('.answercell').each(function() {
```

Comment:
using jquery when selecting elements and loops like this is (very slightly) slower than just using vanilla JS, mainly because jquery does a few other things internally so it can work in more cases.

I think we should stick with the vanilla JS code that was here before for this, even though we might not notice the impact but it does add up! (see #391)

Reply:
we could also go back to the vanilla JS loop for the `commentUsers` one

Reply:
Would a `for` loop work here?